src/app/components/pager/terra-pager.component.html

```diff
@@ -51,7 +51,7 @@
                               [inputIsDisabled]="inputPagingData.totalsCount == 0"
                               [inputIsSmall]="true"
                               [(ngModel)]="inputDefaultPagingSize"
-                              (outputValueChanged)="onChangeOffsetTo($event)">
+                              (ngModelChange)="onChangeOffsetTo($event)">
```

Comment:
Diese Änderungen hat Auswirkungen auf den Typ von `$event`. Dieses ist nun vom Typ `number`. Das müsste ggf. auch in der ts-Datei angepasst werden.

Reply:
done